v2/cmd/asoctl/cmd/export_template.go

```diff
@@ -64,13 +72,20 @@ asoctl export template --version v2.6.0 --crd-pattern "resources.azure.com/*;con
 				uri = template.URIFromVersion(options.version)
 			}
 
+			log := CreateLogger()
```

Comment:
What stream of stderr/stdout does this go to?

If it goes to stdout we can't have it at all (because we need to be able to pipe and this will break piping), if it goes to stderr it's OK but maybe should be behind a `--debug` flag, with the command producing no output by default?

Reply:
Logging goes to `stderr`; this is the same logging used by `import azure-resource` which supports piping the results to a file even though it has lots of progress updates beforehand.